src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.html

```diff
@@ -0,0 +1,26 @@
+<ng-template #usageTooltipTpl>
+  <div [innerHtml]="'<table>' +
```

Comment:
You can remove this `div` and all the strings. Just use a normal `table` element and use `{{ }}` on the variables.